pkgs/tools/archivers/undmg/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, zlib, bzip2
+}:
+
+stdenv.mkDerivation {
+  name = "undmg";
+
+  src = fetchgit {
+    url = https://github.com/matthewbauer/undmg.git;
+    sha256 = "0f2hhb85nwva1grj18q3rib48rgscavnrzqp5mdwxwrywqm9q4pl";
+  };
+
+  buildInputs = [ zlib bzip2 ];
+
+  setupHook = ./setup-hook.sh;
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    chmod +x undmg
```

Comment:
Do you think adding a `install` target to the `Makefile` of `undmg` makes sense here @matthewbauer?